### PR TITLE
[CI] Add github actions: eslint-check

### PR DESCRIPTION
**Pull request by winnd**
1. [UI] Eslint: Fix eslint warnings and errors before use github-actions-eslint-check
2. [CI] Add github actions: eslint-check

### CODE REVIEW

**Comment by winnd** [comment]
It seems still has some steps with how to set github actions run automatically.↓↓

https://stackoverflow.com/questions/58654530/how-to-reject-a-pull-request-if-tests-are-failed-github-actions

**Comment by winnd** [comment]
I will do some optimization rather than `npm install`

File: .github/workflows/eslint-check.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    branches: [ main ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+
+      - name: install dependencies
```
Comment (RobertIndie) [inline .github/workflows/eslint-check.yml:15]:
```suggestion
      - name: Install dependencies
```

File: .github/workflows/eslint-check.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        run: yarn install
+        working-directory: front-end
+
+      - name: run eslint
```
Comment (RobertIndie) [inline .github/workflows/eslint-check.yml:19]:
```suggestion
      - name: Run eslint
```

**Review by RobertIndie — CHANGES_REQUESTED** [review]

**Review by RobertIndie — APPROVED** [review]
LGTM

**Pull request merged**